exonum_launcher/configuration.py

```diff
@@ -90,9 +96,12 @@ def __init__(self, data: Dict[Any, Any]) -> None:
         for (name, value) in data["instances"].items():
             artifact = self.artifacts[value["artifact"]]
             instance = Instance(artifact, name, value.get("config", None))
-            instance.deadline_height = _get_specific("deadline_height", value, parent=data)
+            instance.deadline_height = _get_specific("deadline_height", value, parent=data) or 0
```

Comment:
I think instances shouldn't anymore have this field?

Reply:
Yes, I guess